Course/Course/Data/CourseRepository.swift

```diff
@@ -230,7 +230,7 @@ class CourseRepositoryMock: CourseRepositoryProtocol {
     
     func getCourseDates(courseID: String) async throws -> CourseDates {
         do {
-            let courseDates = try courseDatesJSON.data(using: .utf8)!.mapResponse(DataLayer.CourseDates.self)
+            let courseDates = try CourseRepository.courseDatesJSON.data(using: .utf8)!.mapResponse(DataLayer.CourseDates.self)
```

Comment:
do we need `// swiftlint:disable all` above if you moved json string out this file?